urlwatch

```diff
@@ -109,11 +111,11 @@ class NullHandler(logging.Handler):
 
 log.addHandler(NullHandler())
 
-ERROR_MESSAGE_URLS_TXT = """
-Error: You need to create a urls.txt file first.
+ERROR_MESSAGE_URLS = """
+Error: You need to create either a urls.yaml or a urls.txt file first.
```

Comment:
I'm fine with that. 
My suggestion:
Try to convert urls.txt to urls.yaml, if no urls.yaml exists. To be one the safe side keep urls.txt, but rename it to urls.txt.imported.

In the case there both (urls.txt and urls.yaml) give a warning suggesting the user should import manually. That way we avoid having to merge urls.txt into an existing urls.yaml.